salt/modules/hadoop.py

```diff
@@ -88,6 +88,22 @@ def dfs(command=None, *args):
         return 'Error: command must be provided'
 
 
+def dfsadmin(command=None, *args):
+    '''
+    Execute a command on DFS
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' hadoop.dfsadmin -report
+    '''
+    if command:
+        return _hadoop_cmd('dfsadmin', command, *args)
+    else:
+        return 'Error: command must be provided'
```

Comment:
Is there a reason you're doing the check this way, rather than just relying on `command` instead of `command=None`?

Reply:
I change the command,please check it !